shared/uavobjectdefinition/systemident.xml

```diff
@@ -12,5 +12,6 @@
         <telemetrygcs acked="false" updatemode="manual" period="0"/>
         <telemetryflight acked="false" updatemode="onchange" period="1000"/>
         <logging updatemode="manual" period="0"/>
+	<field name="HoverThrottle" units="% / 100" type="float" elements="1" defaultvalue="0"/>
```

Comment:
Should go with the other fields, also indentation is mixed throughout the file but you can ignore that if you want.